docs/platform/howto/byoc/create-custom-cloud.md

```diff
@@ -769,6 +786,12 @@ Your new custom cloud is ready to use only after its status changes to
            --byoc-id "CUSTOM_CLOUD_IDENTIFIER" >| "tf_dir/tf_file.tf"
          ```
 
+         where
+
+         - `organization-id` is the identifier of your Aiven organization that is to be
+           connected with your own cloud account to create the custom cloud.
+         - `byoc-id` is the identifier of your custom cloud.
```

Comment:
Extracted from the output of the previous `avn byoc create` command: `018b6442-c602-42bc-b63d-438026133f60`